src/ui/CozyDialogs/ConfirmDialog.tsx

```diff
@@ -61,14 +98,56 @@ export const ConfirmDialog = ({
 
         <View style={styles.content}>
           {typeof content === 'string' ? (
-            <Typography>{content}</Typography>
```

Comment:
the text is lighter than before here, is it intended? 
![image](https://github.com/cozy/cozy-flagship-app/assets/12577784/2d2cde8a-4732-485a-8ab0-be436632aa80)